### PR TITLE
Feature gitignore (and some additional link).

### PR DESCRIPTION
**Pull request by TB-EIT**
We include some standard PR-related checks:
* I did this thinh
* I did another thing
* This is related to this feature

### CODE REVIEW

**Comment by BasiukTV** [comment]
I'm not sure what is this PR related to? Can you please clarify?

**Review by BasiukTV — CHANGES_REQUESTED** [review]
Please address the requested changes.

File: empty_folder/.gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+# Empty .gitignore file to commit the folder structure
```
[the remark this answers is not in the document]
Comment (TB-EIT) [reply empty_folder/.gitignore:1]: Same.

**Comment by TB-EIT** [comment]
@BasiukTV please review again. I addressed your feedback.

**Review by BasiukTV — APPROVED** [review]
LGTM

**Comment by BasiukTV** [comment]
@TB-EIT LGTM. Go ahead and merge in.

**Pull request merged**